src/spyglass/common/common_lab.py

```diff
@@ -192,18 +192,19 @@ def create_new_team(
 
         member_list = []
         for team_member in team_members:
-            # LabMember.insert_from_name(team_member)
-            query = (
-                LabMember.LabMemberInfo() & {"lab_member_name": team_member}
-            ).fetch("google_user_name")
+            LabMember.insert_from_name(team_member)
```

Comment:
but `decompose_name` cannot handle `First Last`:

![image](https://github.com/khl02007/spyglass/assets/9277197/92f45459-d947-45fb-8fa6-e632e2211847)

Reply:
This is a case of sending the result form back to `decompose_name`. My edit on L69 should prevent this when launched via file ingestion, by using the inverted form. Did you rerun your file ingestion with this branch?

Reply:
Yes, that prevents this error because the name is sent as `last, first`. But my point is: what if the user calls this function (`create_new_team`) independently of the nwb file ingestion and passes name as `first last`? Maybe I'm confused but doesn't `create_new_team` call `LabMember.insert_from_name` which calls `decompose_name`?

Reply:
Under the current `decompose_name` structure, it will always be the case that user-passed names with spaces will need to conform to the inverted structure - I don't think there's a way to accept a name as a single string that would correctly decompose all cases of spaces in first and last names.

If we were open to revising the assumptions of the `LabMember` table, I would enforce comma as the delimiter in all cases - but, accepting the status quo, where 'result' is stored as ordered, we have to enforce some delimiter

Reply:
👍